access-modifier-checker/src/main/java/org/kohsuke/accmod/impl/Checker.java

```diff
@@ -444,7 +455,6 @@ public void visitFieldInsn(int opcode, String owner, String name, String desc) {
                     }
                     break;
             }
-            super.visitFieldInsn(opcode, owner, name, desc);
```

Comment:
Why?

Reply:
* It does nothing
* `super` is not used in any of the other override methods.
* It would have to duplicated for both method exit points.